documentation/source/tutorials/pages/computing_metrics_for_registered_images.rst

```diff
@@ -0,0 +1,109 @@
+
+Next, we will compute CSA for each individual vertebral level (rather than averaging) by using the ``-perlevel`` argument.
+
+.. code:: sh
+
+   sct_process_segmentation -i t2_seg.nii.gz -vert 3:4 -vertfile ./label/template/PAM50_levels.nii.gz -perlevel 1 -o csa_perlevel.csv
+
+This command generates a csv file named ``csa_perlevel.csv``, which is partially replicated in the table below.
+
+.. csv-table:: CSA values computed for C3 and C4 vertebral levels
+   :file: csa_perlevel.csv
+   :widths: 15, 9, 14, 8, 7, 7, 7, 7, 7, 7, 7
+   :header-rows: 1
+
+Per-slice CSA
+=============
+
+Finally, to compute CSA for individual slices, set the ``-perslice`` argument to 1, combined with the ``-z`` argument to specify slice numbers (or a range of slices).
+
+.. code:: sh
+
+   sct_process_segmentation -i t2_seg.nii.gz -z 30:35 -perslice 1 -o csa_perslice.csv
+
+This command generates a csv file named ``csa_perslice.csv``, which is partially replicated in the table below.
+
+.. csv-table:: CSA values across slices 30 to 35
+   :file: csa_perslice.csv
+   :widths: 13, 9, 12, 8, 7, 7, 7, 9, 7, 9, 7
+   :header-rows: 1
+
+----------
+
+Analysis of the spinal cord shape
+*********************************
+
+The csv files generated by ``sct_process_segmentation`` also include metrics to analyse the shape of the spinal cord in the axial plane, such as ellipticity, antero-posterior and right-left dimensions. These are of particular interest for studying cord compression. See [Martin et al. BMJ Open 2018] for an example application in degenerative cervical myelopathy.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/sct_process_segmentation-shape-metrics.png
+   :align: center
+   :figwidth: 60%
+
+   Additional shape metrics computed by ``sct_process_segmentation``.
+
+----------
+
+Verifying the correctness of the metrics
+****************************************
+
+You may have noticed that the csv files generated by ``sct_process_segmentation`` also include the angles between the cord centerline and the normal to the axial plane. Angle_AP corresponds to the angle about the AP axis, while angle_RL corresponds to the angle about the RL axis. These angles are used to correct for the tilt of the spinal cord relative to the superior-inferior axis. Therefore, if you obtain inconsistent CSA values, it is a good habit to verify that the values of these angles are reasonable.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/csa-angles.png
+   :align: center
+   :figwidth: 50%
+
+   Visualization of CSA, as well as the angles (AP, RL) used to compute the CSA.
+
+----------
+
+Next: Including additional images in your analysis (MT data)
+************************************************************
+
+The next two tutorials will cover how to take additional images that were acquired during the same session and include them in your analysis.
```

Comment:
```suggestion
The next tutorials will cover how to take additional images that were acquired during the same session and include them in your analysis.
```
in case we add more tutorials